src/lib/Scenes/Artwork/Components/CommercialInformation.tsx

```diff
@@ -3,7 +3,7 @@ import { CommercialInformation_artwork } from "__generated__/CommercialInformati
 import { capitalize } from "lodash"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import { ArtworkExtraLinks } from "./ArtworkExtraLinks"
```

Comment:
This was named this way to match our need to add additional, similar "links" for other flows.

See the BNMO case:
![image](https://user-images.githubusercontent.com/2081340/62388069-5c1a6b80-b52a-11e9-9a1d-bb6e7c68d068.png)

And the auctions case
![image](https://user-images.githubusercontent.com/2081340/62388077-5fadf280-b52a-11e9-86c5-36888125a019.png)

Reply:
Ah I see, I'll name it back to that!